lib/core/unit-tests/test_ink_inet.cc

```diff
@@ -21,14 +21,14 @@
     limitations under the License.
 */
 
-#include <ts/TextView.h>
-#include <ts/BufferWriter.h>
-#include <ts/ink_inet.h>
+#include <tsutil/TextView.h>
```

Comment:
Files that we want to expose to be used by C++ plugins should go into the C++ API.

Then we would have 2 directories for APIs: include/ts and include/tscpp

Reply:
As noted before, I think that's a bad idea because it will confuse headers that are used in the TS core with headers that are used only by plugins. None of the headers in "include/tscpp" should ever be included in any core file. If utility headers are moved there then you'll have to note, somewhere, somehow, _these_ are OK to include in core but _those_ are not. There's also the issue of which library then contains the implementation code, where part of "include/tscpp" is in "libtsutil" and some is in "libtscpp", while conversely "libtsutil" is built for headers from "tsutil" and "tscpp". I don't think that will end well.

Reply:
Also, each header will have to note whether it requires "libtscpp" or "libtsutil", as opposed to a simpler rule of "headers in tscpp require libtscpp, headers in tsutil require libtsutil".

Reply:
I think it is more likely to confuse plugin writers that don't know about two different directories for C++ headers and if they have to link with a library or not.

Reply:
That would seem to lead to linking everything in to the `traffic_server` binary and having plugins not link to any libraries at all. Is that the goal?